### ISSUE TITLE
Make CitraVR be able to used while laying down

### ISSUE THREAD

**Issue by xAstroBoy** [amwatson/CitraVR#13]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

I like how the emulator is coming, but the most best thing would to allow to play it while laying down as well.

### Why would this feature be useful?

So it can be played while in bed

**Comment by jakedowns** [comment]
yeah, maybe we could rename this to "allow repositioning of bottom screen"
